### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by MarcoDiaz2000**
In this project, I added some new tables and added foreign key columns to their existing animal table to model one-to-many relationships. Then, I used JOIN to query the data.

**Screenshot of queries:**

- What animals belong to Melody Pond?
c1

- List of all animals that are pokemon (their type is Pokemon).
c2

- List all owners and their animals, remember to include those that don't own any animal.
c3

- How many animals are there per species? 
c4

- List all Digimon owned by Jennifer Orwell.
c5

- List all animals owned by Dean Winchester that haven't tried to escape.
c6

- Who owns the most animals?
c7

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,5 +1,95 @@
...
+-- How many animals are there per species?
+vet_clinic=# SELECT species.name, COUNT(animals.id)
+vet_clinic-# FROM animals
+vet_clinic-# JOIN species ON animals.species_id = species.id
+vet_clinic-# GROUP BY species.name;
+  name   | count
+---------+-------
+ Pokemon |     5
+ Digimon |     6
+(2 filas)
```
Comment (alexander16108) [inline queries.sql]: the following queries result are incorrect please fix this
- How many animals are there per species?
- List of all animals that are pokemon (their type is Pokemon).

**Comment by addod19** [comment]
### Hi @MarcoDiaz2000, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because your project seems to have been reviewed but I do not find or see the review in the PR

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by MarcoDiaz2000** [comment]
Hi @addod19 
That's right, the previous reviewer forgot to put the revision.

I have already reported in the new review request that the reviewer forgot to make the revision, but what can I do?

That's why I requested another review, but now you say that you can't review.

So what do I do?

p1

When you check, you will see that there are only two participants.

I would also like to know who was the first reviewer.



> ### Hi @MarcoDiaz2000,
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that is invalid because your project seems to have been reviewed but I do not find or see the review in the PR
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by MarcoDiaz2000** [comment]
The first reviewer forgot to put the review.

I have already informed in the new revision request that the reviewer forgot to put the revision. 

In the new request I report this event. 

But now the second reviewer tells me that he cannot review because he cannot find the comments of the first reviewer.

I hope that the next reviewer can help me, and do me the favor of being able to review my project, or advise me on how to proceed.

Thanks

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @MarcoDiaz2000 , 

## my apologies for the inconvinence

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- good pr

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi @MarcoDiaz2000, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Correct Git-flow followed :100: 
- [ ] README file is professional and has all the relevant headers :1st_place_medal: 
- [ ] All query results are correct and supported with screenshots in the PR :rocket: 
- [ ] Data insertion are without any errors :+1: 

### Optional
- [x] It would be nice to remove the `vet_clinic=*#` immitating the terminal in the code


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**